### ISSUE TITLE
Brand new install found 36 vulnerabilities (25 low, 10 moderate, 1 high)

### ISSUE THREAD

**Issue by simonh1000** [roots/sage#2323]
* [ ] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
* [x] This request isn't a duplicate of an existing issue
* [ ] I've read the [docs](https://roots.io/documentation/) and followed them (if applicable)
* [ ] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

## Description

Completely new install and after `npm install` I get this report
`found 36 vulnerabilities (25 low, 10 moderate, 1 high)`

## Steps to reproduce

see https://roots.io/sage/

**Expected behavior:** [What you expect to happen]

no security issues
**Actual behavior:** [What actually happens]
many security issues

**Reproduces how often:** [What percentage of the time does it reproduce?]

## Versions

9.0.9
You can get this information from referencing `CHANGELOG.md`. Also, please include the OS and what version of the OS you're running.

## Additional information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by Julix91** [comment]
First of all, don't be too alarmed: https://www.voitanos.io/blog/don-t-be-alarmed-by-vulnerabilities-after-running-npm-install

> If you really want to be a good developer citizen, you should jump over to the package that takes the dependency, fork their repo, modify the dependency to remove the reference to the vulnerable package version & replace it with the “fixed” version, test to make sure everything still works, then submit a pull request (PR). Once the PR is merged in, other packages who take the dependency on that one will use the new dependency and the vulnerability will go away.

Things like this: https://github.com/roots/sage/pull/1199

here's what needs updating: https://david-dm.org/roots/sage?type=dev .

Cheers,
Julian

**Comment by TheAggressive** [comment]
I've submitted a pull request that would take care of this concern: [#2341](https://github.com/roots/sage/pull/2341)

**Comment by retlehs** [comment]
tracking here: #2466
